app/models/playlist.rb

```diff
@@ -45,6 +47,10 @@ def generate_access_token
     end
   end
 
+  def access_token_url
+    url_helpers.playlist_path(self, only_path: false) + '?token=' + self.access_token
```

Comment:
Instead of bringing the url_helpers into the model, how about moving this method into the playlist_helpers.rb?  I think that makes the most sense.  This is how I'm thinking of it: the access_token is data so it should live on the model while the url is a rendering of that data so it belongs outside the model.